data/mods/MindOverMatter/hobbies.json

```
@@ -0,0 +1,9 @@
+[
+  {
+    "type": "profession",
+    "id": "meditation",
```

```suggestion
    "id": "meditation",
    "name": "Meditation",
    "description": "You make a habit of clearing your thoughts and finding your center, even amidst the chaos of the world.",
```
It looks like name and description are mandatory fields for translation parser, <https://github.com/CleverRaven/Cataclysm-DDA/actions/runs/5352152552/jobs/9706777205?pr=66391>